apache/superset
# chore(explore): Create new entrypoints for Echarts Timeseries

### SUMMARY
This PR adds the new charts/entry points implemented in https://github.com/apache-superset/superset-ui/pull/1249. For more details, see description.
https://github.com/apache-superset/superset-ui/pull/1249 must be merged before this PR! After that, the superset-ui versions need to be bumped

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
See https://github.com/apache-superset/superset-ui/pull/1249

### TESTING INSTRUCTIONS
1. Create Echarts Area Chart. Verify that you can change series type to Line, Smooth Line, Step Start/Middle/End.
2. Create Line, Smooth Line, Bar, Scatter, Step Start/Middle/End Timeseries chart. Verify that you CANNOT change the series type or select "area".
3. Create Timeseries Chart. Verify that all series types and "area" checkbox are available

### ADDITIONAL INFORMATION



@junlincc @villebro
